### DISCUSSION TITLE
Custom folders to organize aliases, with search limited to current folder.

### DISCUSSION THREAD

**Discussion by Dustysa4** [simple-login/app#2185] [Feature Requests]
In addition to a list view (as others have requested), I think it would be useful to have the ability to organize aliases into custom-named folders. Many of us are using an API key within our password managers. Most password managers have the feature to organize logins into custom-named folders. It’d be helpful for this to translate across all platforms such that my finance logins are in a “Finance” folder in my password manager, the corresponding alias is in a “Finance” folder in SimpleLogin, and then those alias emails are in a list view inside the folder so I can easily capture them and create rules to filter them into a “Finance” folder in my legit email account. Just as an example.

**Comment by justadityaraj** [comment]
This will be extremely useful.

**Comment by mwsmws22** [comment]
+1 would love this feature. Then I can organize aliases by password manager
